ngraph/src/ngraph/frontend/onnx_import/onnx.cpp

```diff
@@ -68,20 +101,11 @@ namespace ngraph
                 // Try parsing input as a prototxt message
                 if (!google::protobuf::TextFormat::Parse(&iistream, &model_proto))
                 {
-                    throw detail::error::stream_parse{stream};
+                    throw detail::error::stream_parse{"std::istream"};
```

Comment:
Here we throw because parsing from `IstreamInputStream` failed - shouldn't we use this in the error message?

Reply:
Actually I intended to provide the user information about the source which he used - here it is `std::istream`. The `IstreamInputStream` is our internal temporary object used to provide data in expected form to protobuf parser.